tests/testdata/default_config.yaml

```diff
@@ -264,6 +264,9 @@ train:
   # Allow CPU offload for deepspeed optimizer.
   # Default: False
   deepspeed_cpu_offload_optimizer: false
+  # Pick a distributed training backend framework for GPU accelerated full fine-tuning.
+  # Default: fsdp
+  distributed_training_backend: fsdp
```

Comment:
fsdp as the default? is this the right behavior? I know its what we want to move towards, but not sure if its stable enough yet.

Reply:
i thought about it, seemed like a toss-up either way, looking at the future use cases
changed it back to deepspeed now